### ISSUE TITLE
Data loss caused by move to SMB share

### ISSUE THREAD

**Issue by eandmsz** [fman-community/fman#653]
fman: 1.5.8
OS: kubuntu 19.04

I have tried to move a file to a mounted SMB share where I don't have write permission because the network share was mounted as root (in rw mode), but I was not running fman as root.

Fman warned: "Error copying example.txt (permission denied). Do you want to continue?"
Yes, Yes to all, Abort

I pressed "Yes" and guess what: the file didn't got copied obviously, but it got deleted from the source location.
"Yes to all" can destroy multiple files this way.

I think this is a critical bug as it leads to data loss.

**Comment by mherrmann** [comment]
Very sorry for the trouble. Unfortunately, I cannot reproduce this. Here's what I tried:

1. I created a folder owned by `root`.
2. I selected two files in fman.
3. I pressed <kbd>F6</kbd> to move the two files into the folder from step 1.

As expected, fman opened a dialog `Error moving - permission denied. Do you want to continue?`. Like you said, I then pressed `Yes`. This resulted in another `Error moving` alert. However! The files were _not_ deleted in the source folder.

Are you sure this is reproducible?

**Comment by eandmsz** [comment]
No problem, I haven't lost anything serious.
Yes it's reproducible, but if you just try to copy a file on a local filesystem where you don't have access then the bug will not occur.

But I have found an easier way to reproduce it (without setting up a test network with Samba and everything):

**Create an ISO file (or download any CD image from somewhere)**
```
touch /tmp/somefile
mkisofs -o /tmp/example.iso /tmp/somefile
```

**Mount the ISO file as loop device (will be mounted as readonly)**
```
mkdir /mnt/testdir
sudo mount -o loop /tmp/example.iso /mnt/testdir
```

**Test it:**
```
touch /tmp/testfile
mv /tmp/testfile /mnt/testdir
```

 ...it will complain that the filesystem is read-only, but the testfile will remain in place, however if you try to move the testfile with fman, it will get deleted

**Comment by mherrmann** [comment]
Fixed in [fman 1.5.9](https://fman.io/changelog#v1.5.9). Sorry again for the trouble.
